models.py

```diff
@@ -108,7 +106,7 @@ def run(self):
 
     def countdown(self, exc_time):
         for sec in range(exc_time):
-            self.logger.info("{0} seconds until the next exercise".format(exc_time-sec))
+            self.logger.debug("{0} seconds until the next exercise".format(exc_time-sec))
```

Comment:
Yeah, I've waffled on fixing this. _Really_ what I think we'd want is a little pop up with something like:
* Workout
* Time left
* Next Exercise
* (Maybe total time left?)